src/routes/index.ts

```diff
@@ -119,6 +125,14 @@ router.post(config.MANAGING_OFFICER_CORPORATE_URL, authentication, navigation.ha
 router.post(config.MANAGING_OFFICER_CORPORATE_URL + config.ID, authentication, navigation.hasBeneficialOwnersStatement, ...validator.managingOfficerCorporate, checkValidations, managingOfficerCorporate.update);
 router.get(config.MANAGING_OFFICER_CORPORATE_URL + config.REMOVE + config.ID, authentication, navigation.hasBeneficialOwnersStatement, managingOfficerCorporate.remove);
 
+//  FETCH AND UPDATE COMPANY PROFILE
+router.get(config.CONFIRM_OVERSEAS_COMPANY_PROFILES_URL, authentication, confirmOverseasCompanyDetails.get);
```

Comment:
It should be CONFIRM_OVERSEAS_ENTITY_URL. It's not an overseas company and I'd like to avoid that confusion.